Userland/Applications/SystemMonitor/main.cpp

```diff
@@ -254,18 +255,31 @@ int main(int argc, char** argv)
         return pid_index.data().to_i32();
     };
 
+    auto selected_name = [&](ProcessModel::Column column) -> String {
+        if (process_table_view.selection().is_empty())
+            return {};
+        auto pid_index = process_table_view.model()->index(process_table_view.selection().first().row(), column);
+        return pid_index.data().to_string();
+    };
+
     auto kill_action = GUI::Action::create(
         "&Kill Process", { Mod_Ctrl, Key_K }, { Key_Delete }, Gfx::Bitmap::try_load_from_file("/res/icons/16x16/kill.png").release_value_but_fixme_should_propagate_errors(), [&](const GUI::Action&) {
             pid_t pid = selected_id(ProcessModel::Column::PID);
-            if (pid != -1)
+            if (pid == -1)
+                return;
+            auto rc = GUI::MessageBox::show(window, String::formatted("Do you really want to kill \"{}\" (PID {})?", selected_name(ProcessModel::Column::Name), pid), "SystemMonitor", GUI::MessageBox::Type::Question, GUI::MessageBox::InputType::YesNo);
```

Comment:
While the program name itself is `FooBar`, visually we always stylize it as `Foo Bar` - in window titles, application shortcuts, etc.

```suggestion
            auto rc = GUI::MessageBox::show(window, String::formatted("Do you really want to kill \"{}\" (PID {})?", selected_name(ProcessModel::Column::Name), pid), "System Monitor", GUI::MessageBox::Type::Question, GUI::MessageBox::InputType::YesNo);
```

Reply:
Fixed